tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/PulsarContainer.java

```diff
@@ -97,9 +98,13 @@ public void start() {
         this.withCreateContainerCmdModifier(createContainerCmd -> {
             createContainerCmd.withHostName(hostname);
             createContainerCmd.withName(getContainerName());
-            createContainerCmd.withEntrypoint(serviceEntrypoint);
+            createContainerCmd.withEntrypoint(serviceEntryPoint);
         });
 
+        if(Boolean.getBoolean("containerLogs")) {
+            this.withLogConsumer(new Slf4jLogConsumer(log).withPrefix(hostname));
```

Comment:
frankly speaking, I am not sure what do you actually get from using `Slf4jLogConsumer` here. because in pulsar test image, the pulsar logging is not dumped directly to stdout or stderr, it is actually piped to a log file under `/var/log/pulsar`. so when you enable `Slf4jLogConsumer` here, you are dumping the log from supervisord, not the actual pulsar log, which it is usually not useful.